workflows/bacterial_genomics/amr_gene_detection/amr_gene_detection.ga

```diff
@@ -0,0 +1,683 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "Analysis of assembly genome to provide antimicrobial gene detection",
```

Comment:
how about: "Antimicrobial resistance gene detection from assembled bacterial genomes"?

Reply:
Thanks for the suggestion !